wiki.moinmoin

```diff
@@ -316,27 +525,49 @@ TODO: - does not deal with system authentication (eg, pam), etc)
 TODO: Problems:
 
 [Common blockers]
+RULE: - There are plenty of testing requirements, hopefully already resolved
+RULE:   by the reporter upfront, see "Quality assurance - testing" section of
+RULE:   the Main Inclusion requirements
+
 OK:
 TODO: - does not FTBFS currently
 TODO: - does have a test suite that runs at build time
 TODO:   - test suite fails will fail the build upon error.
 TODO: - does have a non-trivial test suite that runs as autopkgtest
-TODO: - if special HW does prevent build/autopkgtest is there a test plan, code, log provided?
+TODO: - if special HW does prevent build/autopkgtest is there a test plan, code,
+TODO:   log provided?
 TODO: - if a non-trivial test on this level does not make sense (the lib alone
 TODO:   is only doing rather simple things), is the overall solution (app+libs)
 TODO:   extensively covered i.e. via end to end autopkgtest ?
-TODO: - no translation present, but none needed for this case (user visible)?
-TODO: - not a python/go package, no extra constraints to consider in that regard
 TODO: - no new python2 dependency
-TODO: - Python package that is using dh_python
-TODO: - Go package that uses dh-golang
+TODO: - Python package, but using dh_python
+TODO: - Go package, but using dh-golang
 
 TODO: Problems:
 
 [Packaging red flags]
+RULE: - Does Ubuntu carry a non necesary delta?
+RULE: - If it's a library, does it either have a symbols file or use an empty
+RULE:   argument to dh_makeshlibs -V?  (pass such a patch on to Debian, but
+RULE:   don't block on it).  Note that for C++, see [[DailyRelease/FAQ]] for a
+RULE:   method to demangle C++ symbols files.
+RULE: - Does it have a watch file? (If relevant, e.g. non-native)
+RULE: - Is its update history slow or sporadic?
+RULE: - Is the current release packaged?
+RULE: - Will entering main make it harder for the people currently keeping it
+RULE:   up to date?  (i.e. are they only MOTUs?)
```

Comment:
double space (ok, I stop mentioning those, but there are some below and above) :))